VideoWeb/VideoWeb/ClientApp/src/app/services/events.service.ts

```diff
@@ -210,6 +212,24 @@ export class EventsService {
             }
         );
 
+        this.connection.on('ParticipantRemoteMuteMessage', (participantId: string, conferenceId: string, isRemoteMuted: boolean) => {
+            this.logger.debug('[EventsService] - Participant Remote mute status change received: ', {
+                participantId,
+                conferenceId,
+                isRemoteMuted
+            });
+            this.participantRemoteMuteStatusSubject.next(isRemoteMuted);
```

Comment:
we dont need the participant or conference because the even will only come through to the relvant participants